### PR TITLE
Issue 5. Comments for functions: remove diagonal, remove negative wei…

### PR DESCRIPTION
**Pull request by egolol**
…ghts, symmetrize.

### CODE REVIEW

**Comment by giovannivolpe** [comment]
I've commented remove_diagonal.
Please, comment the other two files.

**Comment by giovannivolpe** [comment]
Invert also in remove_diagonal so that you start with the simplest syntaxis

**Comment by egolol** [comment]
New update, please review.  If you go to file changed section you can add inline comments to code.

File: braph/graph/remove_negative_weights.m [hunk 1]
```diff
@@ -1,6 +1,26 @@
 function B = remove_negative_weights(A, varargin)
-
-    % 'RemoveNegativeWeightsRule' (input from varargin)
+    % REMOVE_NEGATIVE_WEIGHTS removes the negative values of a matrix
+    %
+    % B = REMOVE_NEGATIVE_WEIGHTS(A) removes the negative weights of a
+    %   matrix A using default RULE 'zero'.
+    %
+    % B = REMOVE_NEGATIVE_WEIGHTS(A, RULE) removes the negative
```
Comment (giovannivolpe) [inline braph/graph/remove_negative_weights.m]: B = REMOVE_NEGATIVE_WEIGHTS(A, 'RemoveNegativeWeightsRule', RULE) 

change also below accordingly

File: braph/graph/remove_negative_weights.m [hunk 2]
```diff
@@ -9,9 +29,9 @@
     end
 
     switch lower(remove_negative_weights_rule)
-        case {'abs', 'absolute', 'modulus'}  % take-absolute-value rule
+        case {'abs', 'absolute', 'modulus'}  %  take-absolute-value rule
             B = abs(A);        
-        otherwise % {'zero', '0', 'null'}  % set-to-zero rule
+        otherwise %  {'zero', '0', 'null'}  set-to-zero rule
```
Comment (giovannivolpe) [inline braph/graph/remove_negative_weights.m]: I understand why you made this change, but I prefer this as it was before.
I can tell you why in person on Monday.

File: braph/graph/symmetrize.m [hunk 3]
```diff
@@ -1,6 +1,29 @@
 function B = symmetrize(A, varargin)
-
-    % 'SymmetrizeRule' (input from varargin)
+    % SYMMETRIZE symmetrizes a matrix
+    %
+    % B = SYMETRIZE(A) symmetrizes the matrix A with default RULE 'max'.s
+    %
+    % B = SYMMETRIZE(A, RULE) symmetrizes the matrix A by the
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: Same comment as above

File: braph/graph/symmetrize.m [hunk 4]
```diff
@@ -13,9 +36,9 @@
             B = A + transpose(A);
         case {'av', 'average'}  % average rule
             B = (A + transpose(A)) / 2;
-        case {'min', 'minimum', 'or', 'weak'}  % minimum rule
+        case {'min', 'minimum', 'or', 'weak'}  %  minimum rule
             B = min(A, transpose(A));
-        otherwise  % {'max', 'maximum', 'and', 'strong'} % maximum rule
+        otherwise  %  {'max', 'maximum', 'and', 'strong'} maximum rule
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: Same as above

**Comment by egolol** [comment]
Updated

**Comment by giovannivolpe** [comment]
Let's merge this together this afternoon when we meet.

File: braph/graph/remove_negative_weights.m [hunk 5]
```diff
@@ -1,6 +1,26 @@
...
+    % B = REMOVE_NEGATIVE_WEIGHTS(A, 'RemoveNegativeWeightsRule, RULE) removes the negative
+    %   weights of a matrix A using the RULE specified.
+    %   Admissible RULE options are:
+    %       RULE    -   'zero' (default) | 'abs' | 'absolute' | 'modulus' 
+    %                   'zero' - convert negative values to zeros.
+    %                   'abs' - convert negative values to its absolute
+    %                           value.
+    %                   'absolute' - convert negative values to its absolute
+    %                           value.
+    %                   'modulus'  - convert negative values to its absolute
```
Comment (giovannivolpe) [inline braph/graph/remove_negative_weights.m]: remove extra space

File: braph/graph/remove_negative_weights.m [hunk 6]
```diff
@@ -9,9 +29,9 @@
     end
 
     switch lower(remove_negative_weights_rule)
-        case {'abs', 'absolute', 'modulus'}  % take-absolute-value rule
+        case {'abs', 'absolute', 'modulus'}  %  take-absolute-value rule
```
Comment (giovannivolpe) [inline braph/graph/remove_negative_weights.m]: revert

File: braph/graph/remove_negative_weights.m [hunk 7]
```diff
@@ -9,9 +29,9 @@
...
             B = abs(A);        
-        otherwise % {'zero', '0', 'null'}  % set-to-zero rule
+        otherwise %  {'zero', '0', 'null'} % set-to-zero rule
```
Comment (giovannivolpe) [inline braph/graph/remove_negative_weights.m]: add spaces

File: braph/graph/symmetrize.m [hunk 8]
```diff
@@ -1,6 +1,29 @@
 function B = symmetrize(A, varargin)
-
-    % 'SymmetrizeRule' (input from varargin)
+    % SYMMETRIZE symmetrizes a matrix
+    %
+    % B = SYMETRIZE(A) symmetrizes the matrix A with default RULE 'max'.s
+    %
+    % B = SYMMETRIZE(A, 'SymmetryzeRule', RULE) symmetrizes the matrix A by the
+    %   Symmetrize RULE specified by the PropertyName.
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: remove leading spaces

File: braph/graph/symmetrize.m [hunk 9]
```diff
@@ -1,6 +1,29 @@
...
+    %   Admissible RULE options are:
+    %       RULE    -   'max' (default) | 'sum' | 'add' | 'av' | 'average' |
+    %                   'min' | 'minimum' | 'or' | 'weak'
+    %                   'max' - maximum between inconnection and outconnection (default)
+    %                   'sum' - sum of inconnection and outconnection
+    %                   'add' - sum of inconnection and outconnection
+    %                   'av'  - average of inconnection and outconnection
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: remove space

File: braph/graph/symmetrize.m [hunk 10]
```diff
@@ -13,9 +36,9 @@
             B = A + transpose(A);
         case {'av', 'average'}  % average rule
             B = (A + transpose(A)) / 2;
-        case {'min', 'minimum', 'or', 'weak'}  % minimum rule
+        case {'min', 'minimum', 'or', 'weak'}  %  minimum rule
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: revert

File: braph/graph/symmetrize.m [hunk 11]
```diff
@@ -13,9 +36,9 @@
...
             B = min(A, transpose(A));
-        otherwise  % {'max', 'maximum', 'and', 'strong'} % maximum rule
+        otherwise  %  {'max', 'maximum', 'and', 'strong'} % maximum rule
             B = max(A, transpose(A));
```
Comment (giovannivolpe) [inline braph/graph/symmetrize.m]: as above

**Comment by egolol** [comment]
Changed Comments

**Pull request merged**